tests/unit_tests/test_time_series.py

```diff
@@ -77,6 +79,25 @@ def test_check_data_orientation():
     )
 
 
+def test_check_data_orientation_unbounded_maxshape(tmp_path):
+    filepath = tmp_path / "test.nwb"
+    with h5py.File(filepath, "w") as file:
+        data = file.create_dataset(
+            "data",
+            data=np.ones((10, 3)),
+            maxshape=(None,3),
+        )
```

Comment:
Recommend using proper `TestCase` `setUp`/`tearDown` structure for cleaning up the temporary file

Reply:
the tmp_path fixture should take care of that